src/games/strategy/triplea/delegate/AirBattle.java

```diff
@@ -554,9 +554,6 @@ public void execute(final ExecutionStack stack, final IDelegateBridge bridge) {
 
         @Override
         public void execute(final ExecutionStack stack, final IDelegateBridge bridge) {
```

Comment:
Is the stack used in other classes implementing IExecutable? If not stack could be removed